docs/lib/graphqlapi/fragments/android/getting-started.md

```diff
@@ -157,22 +155,13 @@ public class MainActivity extends AppCompatActivity {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        AWSMobileClient.getInstance().initialize(getApplicationContext(), new Callback<UserStateDetails>() {
```

Comment:
Is that a problem in other languages' snippets as well? If so we should either fix them too or follow up with owners of those platforms.

Reply:
Good question.  I just checked iOS and JS, and they do not have this same issue.